website/docs/r/key_vault_key_rotation_policy.html.markdown

```diff
@@ -0,0 +1,124 @@
+---
+subcategory: "Key Vault"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_key_vault_key_rotation_policy"
+description: |-
+  Manages a Key Rotation Policy.
+---
+
+# azurerm_key_vault_key_rotation_policy
+
+Manages a Key Rotation Policy.
+
+## Example Usage
+
+```hcl
+data "azurerm_client_config" "current" {}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_key_vault" "example" {
+  name                       = "examplekeyvault"
+  location                   = azurerm_resource_group.example.location
+  resource_group_name        = azurerm_resource_group.example.name
+  tenant_id                  = data.azurerm_client_config.current.tenant_id
+  sku_name                   = "premium"
+  soft_delete_retention_days = 7
+
+  access_policy {
+    tenant_id = data.azurerm_client_config.current.tenant_id
+    object_id = data.azurerm_client_config.current.object_id
+
+    key_permissions = [
+      "Create",
+      "Get",
+      "Purge",
+      "Recover"
+    ]
+
+    secret_permissions = [
+      "Set",
+    ]
+  }
+}
+
+resource "azurerm_key_vault_key" "generated" {
+  name         = "generated-certificate"
+  key_vault_id = azurerm_key_vault.example.id
+  key_type     = "RSA"
+  key_size     = 2048
+
+  key_opts = [
+    "decrypt",
+    "encrypt",
+    "sign",
+    "unwrapKey",
+    "verify",
+    "wrapKey",
+  ]
+}
+
+resource "azurerm_key_vault_key_rotation_policy" "example" {
+  key_vault_id = azurerm_key_vault_key.generated.id
```

Comment:
I think this is meant to be `azurerm_key_vault.example.id`? Does it make sense to just combine the `key_vault_id` and `key_name` into one `key_resource_id`/`key_resource_versionless_id` property?